README.md

```diff
@@ -39,14 +39,6 @@ https://annu12340.github.io/Javascript-Notes/
 
 ##Infinities
```

Comment:
Also the markdown is not rendered properly. Could you correct that

(the headings and bold styling (line 79 ) is not showing properly)
![Screenshot](https://user-images.githubusercontent.com/43414928/94775253-1a5eac80-03dd-11eb-8462-d498351e2945.png)

Reply:
Sure

Reply:
Done